carbon-design-system/carbon-addons-iot-react
# fix(hierarchy-list): maintain list search state on drag and drop

Closes #2917 

**Summary**

- Previously, if you searched the list, and then drag-and-drop the list would reset

**Change List (commits, features, bugs, etc)**

- call search handler from handleMove

**Acceptance Test (how to verify the PR)**

- Go to the HierarchyList with large number of items story
- change pageSize to small
- search for 5
- drag 45 to the top of the list
- the search state should be maintain and 45 is now at the top of the list.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- no adverse effects to other stories



**Things to look for during review**
